packages/uniforms/src/useField.tsx

```diff
@@ -112,7 +113,8 @@ export function useField<
     ...props,
     label,
     name,
-    placeholder,
+    // TODO: Should we assert `typeof placeholder === 'string'`?
```

Comment:
I don't think so, a placeholder in Antd is of the `ReactNode` type, so we shouldn't assert it so tight.

Reply:
That's true, however, widening `placeholder` from `string` to` ReactNode` will require casts or asserts in all HTML fields.